engine/src/flutter/lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -2595,14 +2595,43 @@ class EngineSemanticsOwner {
     return (isConsistent, message.toString());
   }
 
+  /// Whether the semantics DOM tree is being updated right now.
+  ///
+  /// Some DOM events can be triggered synchronously with the DOM mutations,
+  /// such as the DOM "focus" event. Handlers of such events may wish to be
+  /// aware of the fact that semantics update is currently in progress. This is
+  /// espectially true for DOM event handlers that send notifications back to
+  /// the framework. As far as the framework is concerned, the semantics tree is
+  /// being updated as part of rendering a frame, as the last phase that follows
+  /// rendering the [Scene] into a [FlutterView]. The framework does not
+  /// anticipate to receive any events that lead to widget state changes that
+  /// alter the current frame's contents. That must be done in the next frame.
+  ///
+  /// DOM event handlers whose notifications to the framework result in state
+  /// changes may want to delay their notifications, e.g. by scheduling them in
+  /// a timer.
+  bool get isUpdatingSemanticsTree => _isUpdatingSemanticsTree;
```

Comment:
I'm guessing this is leftover from a previous iteration? It's not being used anywhere.